src/contents/all/components/knot/Knot/Knot.tsx

```diff
@@ -1,30 +1,113 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import Moment from 'react-moment';
+import { Timeline, Popconfirm, DatePicker, Tooltip } from 'antd';
 
-import { Timeline } from 'antd';
+import { Log, Knot as KnotType } from '../../../global/types';
+
+import KnotInput from '../KnotInput/KnotInput';
 
 import './Knot.scss';
 import '../Trix.scss';
 
-const Knot = props => (
-  <Timeline.Item className={`knot ${props.isLast ? "ant-timeline-item-last" : ""}`}>
-    <div className={`knotCard ${props.knot.private ? "private" : ""}`}>
-      <div className="trix-content" dangerouslySetInnerHTML={{__html: props.knot.body}} />
-    </div>
-    <div className="knotMeta">
-      <Moment
-        filter={(value) =>
-          value.replace(/^a few seconds ago/g, 'just now')
-          .replace(/^a /g, '1 ')
-          .replace(/^an /g, '1 ')
-          .replace("minute", 'min')
-        }
-        fromNow
+// <div className="knotEditorWrapper">
+//   <div className="knotEditor" ref={knotEditor}>
+//     {show &&
+//       <RichTextEditor
+//         autoFocus
+//         toolbarConfig={{ display: [] }}
+//         value={body}
+//         onChange={onChange}
+//         placeholder={hasKnots ? "Add another note..." : "Add a note..."}
+//       />
+//     }
+//   </div>
+// </div>
```

Comment:
Let's get rid of these commented out code blocks since we don't need them anymore